source/reference/limitations.txt

```diff
@@ -81,13 +81,8 @@ Unsupported Collection Types
 Sharded Clusters
 ----------------
 
-- The :ref:`shard topologies <sharding-introduction>` must be the same
-  in the source and target clusters. The following configurations are
-  not supported.
-
-  - Replica set to sharded cluster.
-  - Sharded cluster to replica set.
-  - Unequal numbers of source and destination shards.
+- ``mongosync`` does not support synchronization from a sharded cluster
+  to a replica set.
```

Comment:
There is 2 additional limitations that affect the RS->SC mode (I think I might have added them on [Mongosync Downstream Changes](https://docs.google.com/document/d/1vDjeXw8x592d3NLKL5RHL6Nouw64heAL-yod_PyqKME/edit#) concurrently with your work, sorry):
* For collections the user included in the sharding list, it is undefined behavior to rename them **across databases** (on source) **at any point** while Mongosync is running. For context: this is due to limitations in renaming collections on sharded clusters (here, destination), see `Starting in MongoDB 5.0, you can use the renameCollection command to change the name of a sharded collection. The target database must be the same as the source database.` in [renameCollection documentation](https://www.mongodb.com/docs/manual/reference/command/renameCollection/#sharded-collections).
* If they opt-in to `createSupportingIndexes: true`, they are not allowed to create that index afterward on source until Mongosync is stopped: either they create that index on source before Mongosync starts or after the migration is done and Mongosync is stopped.

Maybe those needs to also be mentioned in other places in the documentation ?

Reply:
Fixed